tests/e2e-ui-tests/data/page_constants.js

```diff
@@ -1,25 +1,28 @@
 module.exports = {
```

Comment:
One thing we could eventually do is move these page values into their own page objects, but that is an improvement we can do in the future